doc/src/atomvm-tooling.md

```diff
@@ -210,9 +210,9 @@ You can now use a serial console program such as [minicom](https://en.wikipedia.
     INFO [74] AtomVM: Exited with return: ok
     INFO [78] AtomVM: AtomVM application terminated.  Going to sleep forever ...
 
-#### Raspberry Pico
+#### Raspberry Pi Pico and Pico 2
```

Comment:
I need just a minor clarification: where do we want to use the term `Raspberry Pi Pico` and where rp2040?

Reply:
Thank you. I suggest to use "Raspberry Pi RP2" when referring to the soc family (RP2040 and RP2350) and Pico, Pico-W and Pico 2 when referring to the boards. I updated the documentation following this approach.